src/uniswap-summaries.md

```diff
@@ -3449,6 +3511,7 @@ module SOLIDITY-UNISWAP-SUMMARIES
   imports SOLIDITY-UNISWAP-GETAMOUNTIN-SUMMARY
   imports SOLIDITY-UNISWAP-PAIRFOR-SUMMARY
   imports SOLIDITY-UNISWAP-FIDSWAP-SUMMARY
+  imports SOLIDITY-UNISWAP-SWAP-SUMMARY
```

Comment:
Is there any reason to put it here instead of in the end of the list?

Reply:
I just put it there because its name was similar to `fidSwap`, but no particular reason.
Another is that when everything is put in the end, it leads to conflicts, but this way they are avoided!

Reply:
Perfect! And good to know!